### DISCUSSION TITLE
DayPicker does not automatically navigate to selected date

### DISCUSSION THREAD

**Discussion by tyforcode** [gpbl/react-day-picker#1992] [Support]
### Bug description

Thanks for the library! I found an issue where if you set the `selected` value of the `<DayPicker>` component with something like an initial state, the component will not show the selected date in the UI. Instead, it will default to the earliest date in the range. 

### To reproduce

https://codesandbox.io/p/sandbox/react-daypicker-forked-4n69dz

#### Steps

1. Step an initial `Date` state
2. Set the following props on the `DayPicker` component
```
        fromDate={new Date("10/10/2020")}
        toDate={new Date("02/02/2022")}
        captionLayout="dropdown"
```
3. Observe the component or refresh the sample to demonstrate the bug

### Expected behavior

Ideally, the `DayPicker` component would show the `selected` value by default when it's set by an outside state. 

### Screenshots

n/a

### Additional context

My specific use case is around a combination of `fromDate`, `toDate` and `captionLayout` where the year does not change correctly. However, this is also applicable to the base `DayPicker` component.

**Comment by gpbl** [comment]
Hi @tyforcode! Thanks for your feedback, I understand why this is unexpected. It could be enough to improve the documentation, as DayPicker in this case works as intended.

To set the default month when rendering DayPicker use the `defaultMonth` prop:

https://react-day-picker.js.org/basics/navigation#change-the-default-month

Example:

https://codesandbox.io/p/sandbox/react-daypicker-forked-js8683?file=%2Fsrc%2FApp.tsx%3A1%2C1-24%2C1
